simvp/datasets/video_dataset.py

```diff
@@ -0,0 +1,232 @@
+# =============================================================================
+# Load Data
+# =============================================================================
+
```

Comment:
Let's add a comment that says something like:

`# Code reference: https://github.com/TUM-LMF/FutureGAN/blob/master/video_dataset.py`